libs/community/langchain_community/document_loaders/parsers/audio.py

```diff
@@ -113,10 +113,12 @@ class OpenAIWhisperParserLocal(BaseBlobParser):
     """
 
     def __init__(
-        self,
-        device: str = "0",
-        lang_model: Optional[str] = None,
-        forced_decoder_ids: Optional[Tuple[Dict]] = None,
+            self,
+            device: str = "0",
+            lang_model: Optional[str] = None,
+            batch_size: int=8,
+            chunk_length_s: int=30,
```

Comment:
variable name is weird, why `_s`?

Reply:
Pushed the wrong file. Please see the following commit.